### PR TITLE
Add support for scheduled jobs with dynamic parameters

### PR DESCRIPTION
**Pull request by rreinoldsc**
# What

Define new event and types for running checkpoints with dynamic parameters

# Why

Required to run as a typed scheduled checkpoint

### CODE REVIEW

File: great_expectations_cloud/agent/actions/run_scheduled_window_checkpoint.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    AgentAction,
+)
+from great_expectations_cloud.agent.actions.run_window_checkpoint import run_window_checkpoint
+from great_expectations_cloud.agent.event_handler import register_event_action
+from great_expectations_cloud.agent.models import (
+    RunScheduledWindowCheckpointEvent,
+)
+
+
+class RunScheduledWindowCheckpointAction(AgentAction[RunScheduledWindowCheckpointEvent]):
```
Comment (shiplet) [inline great_expectations_cloud/agent/actions/run_scheduled_window_checkpoint.py:16]: @rreinoldsc Do we need to add this to one of the `__init__.py` files like we did for the `RunWindowCheckpointAction`?

(see `agent/actions/__init__.py`)

File: tests/agent/actions/test_run_scheduled_window_checkpoint.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    )
+
+
+@pytest.mark.parametrize(
+    "response_kwargs",
+    [
+        pytest.param({"status": 500}, id="non-200"),
+        pytest.param({"json": {"malformed?": "yes"}}, id="malformed JSON"),
+    ],
+)
```
Comment (Shinnnyshinshin) [inline tests/agent/actions/test_run_scheduled_window_checkpoint.py:103]: ♥️

**Review by shiplet — APPROVED** [review]
One question 🚀

**Review by Shinnnyshinshin — APPROVED** [review]
Love it. Really clear implementation @rreinoldsc

File: great_expectations_cloud/agent/actions/run_scheduled_window_checkpoint.py (hunk 1 above)
Comment (rreinoldsc) [reply great_expectations_cloud/agent/actions/run_scheduled_window_checkpoint.py:16]: yes ty, added in 5ce69d9

**Pull request merged**